Dockerfile

```diff
@@ -17,8 +17,6 @@ ADD . .
 
 RUN go get -v -t $(go list ./... | grep -v /vendor/)
 
-RUN go install -v ./buffalo
-
```

Comment:
This is needed. We run our tests using this dockerfile so we want to make sure we are running the tests using the code that was just committed.